qgroundcontrol.qrc

```diff
@@ -247,13 +247,11 @@
     <qresource prefix="/qml">
         <file alias="test.qml">src/test.qml</file>
         <file alias="QmlTest.qml">src/QmlControls/QmlTest.qml</file>
-
```

Comment:
I didn't remove any of these on purpose. It must have been from not this merge but the last one where I had a lot of problems and had to do a lot these by hand. But come to think of it, if that was the case it would not show the differences now. I have no idea... I didn't even look at this file this time around. It was a clean merge and I didn't edit it by hand.